### PR TITLE
Addressing Multiple Issues in PGF and GM Guide

### LINKED ISSUE #104: Need a "Non-Dealer" unifying term
## Need a term for non-dealers that's uniform across ally/enemy NPCs

From #88 comment from @CBroz1, we need a term for non-dealers that isn't as prescriptive as "Bystander" vs. "Minion".

> > I'd prefer to have a unified term for non-dealer so that their allegiance is ambiguous
> 
> A: Are they a dealer?
> B: No, they're a X
> If X could be bystander or minion, saying which gives hints at where the narrative 'should' go.

> I also don't love the asymmetry between bystander and minion. The former sounds passive (which, if DM-NPC or allied combatant, they're not) and the latter sounds weak ('yeah, you can defo oneshot this one') when that may not be the case at higher levels

## Initial proposed fix/suggested resolution

My proposal is that "Minion" should be converted into a creature that is specifically controlled by a Dealer (Player or GM) such as the Necromancer using a Skeleton they control, versus a [INSERT NEW TERM HERE] being of its own free will and mind and indicating they act freely of a Dealer. 

We can stick with NPC as a sweeping term, though I like the flavor of Dealer being card-based if we can find a different term that still has that same feeling/flavor. 

## Post-resolution update steps 

When resolved, we would need to make updates to Player Guides and GM Guide to ensure consistency across all docs.

### LINKED ISSUE #110: Held Actions/Ambushes/Surprise Rounds
### Scenario - Ambush 

Three characters wait around a corner for another to lure the enemy right to them. When that combat is initiated, they would all draw Target Cards and go on that order. 

### Question

How do we create and word the rules for Ambushes or Surprise Rounds where the characters each 'hold' one Major Action that goes off before the top of the first round? And what happens if the 'held' action criteria isn't met (I see an enemy) after 6 seconds, in the event the ambush/lure doesn't go off as planned? Do multiple rounds of combat happen outside of their turn order while they wait, or do they draw TCs to begin combat anyway and then act when the 'trigger' happens still?

### LINKED ISSUE #111: Contested Checks versus Saving Checks "Saves"
## Contested Checks are confusing and need to be cleared up. 

We need to clarify between when a Check is truly Contested (i.e. Tackle) or when someone is making a Saving Check - *Save* - and drawing against their own Target Card. This needs to be explicit in the core ruleset, and then also made clear on Powers as needed.
## Contested Checks vs. Saving Checks

When do the following happen, and how can we write this into the rules to be extremely clear:
1. Contested Check: The initiator draws a Card. This is the Target Card for the Target to aim their Check against. 
- (*Or does the Target simply draw a card against the initiator's TC?)*
3. Saving Check: The initiator spends a resource (FP, a Major Action, etc.) to force target to make a Save. That target draws a Saving Check against their own TC. 

## Alt proposed names for the mechanic (open to others, too)

1. *Save* and *Contest*
2. *Saving Draw* and *Contested Draw*
3. *Saving Check* and *Contested Check*

### LINKED ISSUE #130: Add Holding Breath and Underwater Breathing Rules
## Breathing Underwater and Holding Breath

We should add a section into the PGF regarding Holding Breath. The below wording *Holding Breath* is a placeholder, but we would want to find the right term to add it to the Environment exploration section of the PGF. 

### Add to PGF - Holding Breath Rules

**Holding Breath** - A creature can hold their breath for 1 + N number of additional minutes, where N is the creature's VIT. When this time runs out, or if something causes a creature to no longer hold their breath underwater, the creature begins to Suffocate. At the beginning of their turn, they take 2 damage until they can breathe again. 

Some Powers extend this time limit or prevent a creature from Suffocating.

### Add to GM Guide - Potion of Waterbreathing

Let's add an instance of a magical item removing the above mechanic so there's an example we can lean on for Underwater settings or Adventures requiring extended periods of time underwater. 

**Potion of Waterbreathing** - 1 hour duration - when underwater, a creature under this effect does not need to hold their breath and does not Suffocate when breathing normally underwater.

### LINKED ISSUE #143: Checks against own TC Flaw
Currently we have players maintain a TC and draw against it for certain checks. Unfortunately, this renders critical successes impossible to achieve. As a player, I would be disheartened upon this realization. 

Proposed Change 

Instead of drawing against their own TC, players draw against a TC drawn by the GM. This would include environment driven skill checks and saves. 

For example:
- "Your group moves through the brush in search of the signet. Make a detection check DR 1 against (draws from GM deck), a 2 of clubs.

### PR DESCRIPTION
**Pull request by LockerM**
Addressing the following Issues: 

- [x] Closes #104 
- [x] Closes #110 
- [x] Closes #111 
- [x] Closes #130 
- [x] Closes #143 

Additional minor fixes:
- PGB outdated reference
- Removed Minion Deck for GMs, calling it NPC Deck
- Clarifying character instead of Player on held turns
- Updated Status Conditions to force Saves instead of Checks

### CODE REVIEW

File: docs/src/1_Mechanics/01_PlayerGuide_Full.md [hunk 1]
```diff
@@ -291,6 +290,22 @@ They draw a TC of 3 of Diamonds, permitting a card between A and 6. Jonas draws
...
+Modifier for the relevant domain. A character cannot choose to use a different
+Attribute.
+
+A Save will have a DR determined by the initiator's Skill Modifier used to initiate the
+Save. The DR for a Save is 3 minus half of the initiator's relevant modifier, to a
+minimum of 0. 
+
+Saves may have different effects depending on a success or failure. This can sometimes
+mean a successful Save still results in taking damage or a negative effect. Reference a
+specific Power description for details on the effects.
```
Comment (CBroz1) [inline docs/src/1_Mechanics/01_PlayerGuide_Full.md:307]:
```suggestion
When a character attempts to resist or shrug off a negative effect, they make a Save.
This could be caused by a Power, Status Condition, or a environmental effect. When a
character makes a Save, they draw a card against their own TC and use their relevant Attribute
Modifier, with a DR set by the initiating effect. If the effect is a Power, the DR is
3 minus half of the initiator's relevant Skill modifier. 
Saves may still have consequences even on a success, as specified by the initiating effect.
```

File: docs/src/1_Mechanics/01_PlayerGuide_Full.md [hunk 2]
```diff
@@ -363,6 +380,12 @@ others. If an action specifies a skill, this requires a check.
         - *Hide*: Stealth vs. target(s) Detection; impose Lower Hand on attacks against
          you by target(s) who failed
 
+ A character can choose to hold their Major Action to occur on a stated trigger, such
+ as "I will make an Attack, Weapon when an enemy moves within 1 space of me." If the
```
Comment (CBroz1) [inline docs/src/1_Mechanics/01_PlayerGuide_Full.md:384]: I think this should be stated as 'Weapon Attack' but specified as a power as 'Attack, Weapon' so that attacks get sorted together alphabetically.

```suggestion
 as "I will make a Weapon Attack when an enemy moves within 1 space of me." If the
```

File: docs/src/1_Mechanics/01_PlayerGuide_Full.md [hunk 3]
```diff
@@ -376,39 +399,41 @@ Some Status Conditions will impact a character over time. The duration of each C
...
-a Free Action, a Dealer makes a DR 3 Vitality Check to shake off the effects of the
+a Free Action, a Dealer makes a DR 3 Vitality Save to shake off the effects of the
 poison. On a failure, they discard N+1 cards.
 
 8. **Frozen** Movement speed is reduced by half and all skill checks are drawn at Lower
-Hand. A Dealer makes a DR 3 Strength Check as a Free Action at the end of their turn to
+Hand. A Dealer makes a DR 3 Strength Save as a Free Action at the end of their turn to
 attempt to end the freeze.
 
+9. **Suffocating** At the beginning of their turn, a Dealer makes a DR 2 Vitality Save. On a failure, take 2 damage. On a success, take half damage.
```
Comment (CBroz1) [inline docs/src/1_Mechanics/01_PlayerGuide_Full.md:435]:
```suggestion
9. **Suffocating** At the beginning of their turn, a Dealer makes a DR 2 Vitality Save. On a failure, take 2 damage. On a success, take 1 damage.
```

File: docs/src/1_Mechanics/01_PlayerGuide_Full.md [hunk 4]
```diff
@@ -610,13 +635,23 @@ creature may be covered from one direction but not from another direction.
...
 through *Challenging Terrain*, moving through one space requires 2 speed. While
-standing in Challenging Terrain, all Agility Checks are made with the Lower Hand. 
+standing in Challenging Terrain, all Agility Saves are made with the Lower Hand. 
+
+#### Holding Breath
+
+A character can hold their breath for 1 + N number of additional minutes, where N is the
+character's Vitality Modifier. When this time runs out, or if something causes a
+character to no longer hold their breath while underwater or in a vacuum, the character
+gains the *Suffocating* Status Condition.
```
Comment (CBroz1) [inline docs/src/1_Mechanics/01_PlayerGuide_Full.md:651]: I think we should frame other effects in terms of subtracting from this number.

```suggestion
A character go without air for 1 + 2 times Vitality Mod minutes. 
Taking damage without air reduces this time by 1 minute. 
After this time, the character
gains the *Suffocating* Status Condition.
```

File: docs/src/1_Mechanics/02_GMGuide.md [hunk 5]
```diff
@@ -121,11 +121,9 @@ session to represent allies and enemies. More decks may be useful to
 represent different groups in combat. If you only have one deck available,
 split it into two 26-card decks and shuffle regularly. 
 
-One deck of cards should be designated the *Minion Deck* and one designated the *World
+One deck of cards should be designated the *NPC Deck* and one designated the *World
 Deck*. Different color card backs will help distinguish the two throughout play. The
-Minion Deck is used during combat and when applying negative [Status Conditions](./01_PlayerGuide_Full.md#status-conditions). The World Deck is used to draw Target
-Cards for NPCs, make social checks, and used for drawing a card based on a table out of
-combat. Before each Combat, reshuffle the Minion Deck. You reshuffle the World Deck
+NPC Deck is used to draw Target Cards for NPCs, for their Actions, and when applying negative [Status Conditions](./01_PlayerGuide_Full.md#status-conditions). The World Deck is used to make social checks, and to draw Target Cards representing items and objects in the world. Before each Combat, reshuffle the NPC Deck. You reshuffle the World Deck
```
Comment (CBroz1) [inline docs/src/1_Mechanics/02_GMGuide.md:126]: Suggestions limited to where you've made edits. I'd rephrase here as 
```
The World Deck is used to make social checks, and to draw Target Cards representing 
items and objects in the world. Reshuffle the NPC deck before each Combat. Reshuffle 
the World Deck when empty.
```

File: docs/src/1_Mechanics/02_GMGuide.md [hunk 6]
```diff
@@ -392,6 +390,35 @@ described in the guide or in a Power mechanic.
 4. Arm wrestle: Brute vs. Brute
 5. Impersonation: Initiator Performance vs. Target Detection or (if suspecting) Ingestigation
 
+### Saves
+
+Saves occur when a character is attempting to resist or shrug off a negative effect(see
```
Comment (CBroz1) [inline docs/src/1_Mechanics/02_GMGuide.md:395]:
```suggestion
Saves occur when a character is attempting to resist or shrug off a negative effect (see
```

File: docs/src/1_Mechanics/02_GMGuide.md [hunk 7]
```diff
@@ -392,6 +390,35 @@ described in the guide or in a Power mechanic.
...
+example, a Power may force a character to make an Agility save to avoid being trapped
+in entangling vines. On a failed Save, the character is Entangled. On a successful
+Save, the character avoids the vines and is not Entangled. Other Powers might do 2
+damage on a failed save, or cause half damage (1) on a successful Save. Refer to the
+specific wording of the Power to determine the effect. 
+
+Sometimes, environmental effects may force a character to make a Save. In these cases,
+the GM determines the DR and the relevant Attribute. If an effect would cause a
+character to attempt to resist it to avoid damage or a Status Condition, that character
+should make a Save.
```
Comment (CBroz1) [inline docs/src/1_Mechanics/02_GMGuide.md:417]: 1. I'd avoid repeating across what's here and in the full
2. Why specify 'half' of 2? Why not say 1?

```suggestion
[Full Player Guide](./01_PlayerGuide_Full.md#saves)).

When calling for a Save, choose the Attribute that is most relevant to the
initiating effect. A character cannot choose to use a different Attribute.  
Because characters draw against their own TC, Saves will
never be a Critical Success. You GM can choose to add an additional effect to
a Major Success. 

While all Saves will have effects for failure, some may sill have effects on for success,
such as half damage. Refer to the
specific wording of the initiating effect. 
If an environmental effects initiates a Save, you determine the DR and Attribute as the GM.
If an effect would cause a
character to attempt to resist it to avoid damage or a Status Condition, that character
should make a Save.
```

File: docs/src/1_Mechanics/02_GMGuide.md [hunk 8]
```diff
@@ -954,6 +981,7 @@ create your own Consumables and ask the Community for help as needed.
 | Consumable Name        | Cost  | Effect                                                                                    | Duration |
 | ---------------------- | ----- | ----------------------------------------------------------------------------------------- | -------- |
 | Potion of Healing      | 5 gp  | Regain 1 HP immediately                                                                   |          |
+| Potion of Water Breath | 10 gp | You are immune to the Suffocating Status Condition while underwater                       | 1 hour   |
```
Comment (CBroz1) [inline docs/src/1_Mechanics/02_GMGuide.md:984]: to demonstrate the difference between holding and immunity
```suggestion
| Potion of Water Breath | 10 gp | You are immune to the Suffocating Status Condition                       | 1 hour   |
| Potion of Extended Breath | 5 gp | Your ability to hold your breath is extended by 60 minutes                       | 1 hour   |
```

**Review by CBroz1 — CHANGES_REQUESTED** [review]

File: docs/src/1_Mechanics/01_PlayerGuide_Full.md [hunk 9]
```diff
@@ -181,10 +181,9 @@ the circles around the bullseye.
 
 Once the DR has been announced, players must make the check. Players may consult with
 their GM to determine if another Attribute or Skill may be used instead. A player will
-draw one or more cards. Immediately after drawing, a player will apply any relevant
+draw one card from the top of their deck. Immediately after drawing, a player will apply any relevant
```
[the remark this answers is not in the document]
Comment (D-D-2) [reply docs/src/1_Mechanics/01_PlayerGuide_Full.md]: While as it stands people aren't drawing from anywhere but the top of their deck. But perhaps an effect in the future will let them do so, making the additional words important.

[the remark this answers is not in the document]
Comment (LockerM) [reply docs/src/1_Mechanics/01_PlayerGuide_Full.md]: Reverted the change. I think partly my reasoning here for modifying this is that since this is the first description of making a check in the PGF it may be confusing to a player wondering whether they draw a card, a few cards, whether there's a hand, etc. so I was thinking of finding a way to be very explicit up front that there's a deck of cards and you're top-decking to make a check.

File: docs/src/1_Mechanics/01_PlayerGuide_Full.md [hunk 10]
```diff
@@ -278,9 +277,9 @@ Some checks will be initiated by a character's actions and require others to res
 pitting two skills against each other. This might be one character sneaking past
 another (e.g., Stealth vs. Detection), or one character causing a magical effect that
 another tries to dodge (e.g., Craft vs. Agility). This is a Contested Check. In these
-cases, the character who initiated the action (i.e., the Attacker) reveals a TC and
+cases, the character who initiated the action (i.e., the Attacker)
 sets the DR; the DR for a Contested Check is 3 minus half of the Attacker's relevant
-modifier, to a minimum of 0. The character(s) responding are the Target(s). Rules that
+modifier, to a minimum of 0. The character(s) responding are the Target(s), and must draw a Check against the initiator's TC. Rules that
```
[the remark this answers is not in the document]
Comment (D-D-2) [reply docs/src/1_Mechanics/01_PlayerGuide_Full.md:282]: Same thought here. While it is a maximum of 6 now, I don't know if it'll change in the future. I think "minimum of 0" also helps people tie it all together in their own logic too. It helped me process it from personal experience.

File: docs/src/1_Mechanics/02_GMGuide.md (hunk 5 above)
Comment (LockerM) [reply docs/src/1_Mechanics/02_GMGuide.md]: Updated with your suggestions.

File: docs/src/1_Mechanics/02_GMGuide.md (hunk 7 above)
Comment (LockerM) [reply docs/src/1_Mechanics/02_GMGuide.md]: I updated the text to your suggested copy listed above @CBroz1. 

I had originally wanted to specify 'half' since that will be a common theme in Saves of 'take X damage on a failure or half on a success' but that isn't as important when discussing the high level effects of Saves overall so I'm OK removing that.

**Review by CBroz1 — APPROVED** [review]
